Jukido/Player.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "GameObjects.h"
+
+class Player : public GameObject
+{
+public:
+    Player(sf::Texture* texture) : GameObject(texture) {}
+    void PostInit() override;
+    void update(float deltaTime) override;
+
+    void addRounds(float add) override { m_n_bullets = std::min(m_n_bullets + add * 0.5f, 6.1f); } //6.1 to avoid it capping at 5.99999999999999999
```

Comment:
Can use std::ceil function to round it to biggest integer.
Also, why m-n_bullets is float? It's a number of bullets after all, 5.4 bullets make no sense)